### DISCUSSION TITLE
How to access old logs?

### DISCUSSION THREAD

**Discussion by floydj** [basecamp/kamal#535] [Q&A] [answered]
Trying to do some debugging on an error that occurred a few hours ago. But I've deployed since then, and it seems that the Docker logs only go back to the last deploy. Are the older logs rolled over somewhere for me to access?

**Comment by nickhammond** [comment, marked as answer]
@floydj No, you'll need to stream them somewhere like New Relic, DataDog, Papertrail, Better Stack, ELK, etc.
